### PR TITLE
Outline

### PR DESCRIPTION
**Pull request by rrrutledge**
Looking for feedback from @mrsanz and @utanapishtim

### CODE REVIEW

File: outline.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# An Oriole for Inner Source
+45 min
+
+## Abstract
+Inner source is the application of open source methodologies to internally-developed software.
+While simple to define, inner source can be difficult to explain and implement successfully.
+This presentation introduces the key terms, concepts, and principles for effective inner sourcing along with explanations and real examples.
+It is content that is accessible to newcomers while at the same time refreshing to those with experience.
+The same information in this presentation is also delivered as an O’Reilly Oriole, available freely at https://www.safaribooksonline.com/oriole/.
```
Comment (mrsanz) [inline outline.md:9]: This seems either premature or not relevant anymore.  Remove all references to the Oriole?

File: outline.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+We’ll review the Oriole, how to use it, and other Orioles related to inner source that are on the way.
+
+## Background
+There are many individuals that are excited about inner source.
+They share their experience in [the InnerSource Commons](http://innersourcecommons.org/) and other public locations.
+In my experience it's been reasonably difficult to share things that are directly usable by other inner source practitioners.
```
Comment (mrsanz) [inline outline.md:15]: too vague?  What things?  ideas, examples, terminology, challenges, tools?

File: outline.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Inner source is the application of open source methodologies to internally-developed software.
+While simple to define, inner source can be difficult to explain and implement successfully.
+This presentation introduces the key terms, concepts, and principles for effective inner sourcing along with explanations and real examples.
+It is content that is accessible to newcomers while at the same time refreshing to those with experience.
+The same information in this presentation is also delivered as an O’Reilly Oriole, available freely at https://www.safaribooksonline.com/oriole/.
+We’ll review the Oriole, how to use it, and other Orioles related to inner source that are on the way.
+
+## Background
+There are many individuals that are excited about inner source.
+They share their experience in [the InnerSource Commons](http://innersourcecommons.org/) and other public locations.
```
Comment (mrsanz) [inline outline.md:14]: What other public locations? Is this important?

File: outline.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+## Abstract
+Inner source is the application of open source methodologies to internally-developed software.
+While simple to define, inner source can be difficult to explain and implement successfully.
+This presentation introduces the key terms, concepts, and principles for effective inner sourcing along with explanations and real examples.
+It is content that is accessible to newcomers while at the same time refreshing to those with experience.
+The same information in this presentation is also delivered as an O’Reilly Oriole, available freely at https://www.safaribooksonline.com/oriole/.
+We’ll review the Oriole, how to use it, and other Orioles related to inner source that are on the way.
+
+## Background
+There are many individuals that are excited about inner source.
```
Comment (mrsanz) [inline outline.md:13]: Not sure that you need a whole sentence to share that individuals are excited, consider combining with the next sentence.

File: outline.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+They share their experience in [the InnerSource Commons](http://innersourcecommons.org/) and other public locations.
+In my experience it's been reasonably difficult to share things that are directly usable by other inner source practitioners.
+
+## Goals
+The goal of this presentation is to begin to set guildines, structure, and vocabulary for the industry conversation around inner source.
```
Comment (mrsanz) [inline outline.md:18]: omit "to begin"

Comment (mrsanz) [reply outline.md:18]: for only the industry conversation?  consider omitting "industry"

Comment (mrsanz) [reply outline.md:18]: Whats the difference between `inner source` and `InnerSource`?

**Review by mrsanz — CHANGES_REQUESTED** [review]
wrong doc for review?

File: outline.md (hunk 1 above)
Comment (rrrutledge) [reply outline.md:9]: It's [already gone out](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).  Part of the talk will have to be explaining the switch.  Although I guess there's always the chance to update the abstract (possibly).

File: outline.md (hunk 5 above)
Comment (rrrutledge) [reply outline.md:18]: I used `InnerSource` last year and am using `inner source` this year.

**Comment by rrrutledge** [comment]
Closing for now as I didn't mean to put this one in review (yet).

File: outline.md (hunk 3 above)
Comment (rrrutledge) [reply outline.md:14]: Blogs.  I think so.

File: outline.md (hunk 5 above)
Comment (rrrutledge) [reply outline.md:18]: `The goal of this presentation is to share guidelines, structure, and vocabulary for the conversation around inner source.`

**Pull request closed without merging**